dimensiondata/README.md

```diff
@@ -0,0 +1,64 @@
+# Dimension Data Examples
+Example code that uses jclouds to perform common tasks on an Dimension Data CloudControl. The class names are self-explanatory and the code is well commented for you to follow along.
+
+- [Requirements](#requirements)
+- [Environment](#environment)
+- [The Examples](#examples)
+- [Support and Feedback](#support-and-feedback)
+
+## Requirements
+
+1. Username and password for Dimension Data CloudControl - See the [Getting Started guide](http://jclouds.apache.org/guides/dimensiondata/).
+1. Java Development Kit (JDK) version 6 or later - [Download](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+1. Apache Maven - [Maven in 5 Minutes](http://maven.apache.org/guides/getting-started/maven-in-five-minutes.html).
+1. Git - [Download](http://git-scm.com/downloads).
+
+## Environment
+To setup an environment to compile and run the examples use these commands:
+
+```
+git clone https://github.com/jclouds/jclouds-examples.git
+cd jclouds-examples/dimensiondata/
+```
+
+To download all dependencies, run:
+
+```
+mvn dependency:copy-dependencies "-DoutputDirectory=./lib"
+```
+
+If you also want to download the source jars, run:
+
+```
+mvn dependency:copy-dependencies "-DoutputDirectory=./lib" "-Dclassifier=sources"
+```
```

Comment:
This looks too complicated. Given that you already have an assembly that produces a jar with all required dependencies, the build and run instructions should simply be:
```bash
mvn package
java -jar target/dimensiondata-cloudcontrol-examples-jar-with-dependencies.jar <apiEndpoint> <username> <password>
```